### DISCUSSION TITLE
有人有組過鋼彈模型嗎？

### DISCUSSION THREAD

**Discussion by LeoTsai999** [1101-GS4719-C/Python-Community-GS4719-C#2134] [General]
不知道大家有沒有人組裝過萬代的鋼彈模型？
我個人認為萬代的鋼彈模型在訓練我們的動手能力上非常有幫助！
因此我很推薦大家學習製作鋼彈模型！

**Comment by seastar0687** [comment]
一直對於這方面的領域有所興趣，但由於我涉獵不多，版主可以出一篇文章來帶領我們進入鋼彈模型的世界嗎？
我推薦先從鋼彈的歷史開始介紹，教導我們鋼彈如何起源以及發展成如今的模樣。另外還有鋼彈的分級以及各種相關知識，非常期待版主的下一篇貼文！

**Comment by zhilin93517** [comment]
我認為組裝鋼蛋的過程是非常有趣的，而且可以增加組裝的人的手作能力，如果可以不看說明書自行組裝出來，雖然會很花費時間，但他會強化空間想像的能力，讓你對零件組裝的能力更敏感。

**Comment by NiCL100** [comment]
真心佩服喜歡組裝公仔模型、拼拼圖的人，他們驚人的毅力以及耐心，著實令我讚嘆不已。他們遇到亂麻能靜下心一絲一絲的理，堅持不懈，但我可能會直接快刀斬亂麻甚至視若無睹。老實說，就算有說明書，我還是會組裝錯，而且看到那麼多細小的零件便眼花撩亂，後感頭暈目眩，以致心煩慮亂，旋即收手放棄。這群鍥而不捨的手作家們，令人嘆為觀止！
